tests/cypress/e2e/model-spec.cy.js

```diff
@@ -0,0 +1,128 @@
+
+        // Hide Replication and refreshment 1 and make Replication and refreshment 2 visible
+        assertHiddenAndVisibleToggled('rr-q1', 'rr-q2');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Replication and refreshment 2 and make Operating Environment 1 visible
+        assertHiddenAndVisibleToggled('rr-q2', 'oe-q1');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Operating Environment 1 and make Operating Environment 2 visible
+        assertHiddenAndVisibleToggled('oe-q1', 'oe-q2');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Operating Environment 2 and make Physical Disaster visible
+        assertHiddenAndVisibleToggled('oe-q2', 'pd-q1');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Physical Disaster and make Checksum visible
+        assertHiddenAndVisibleToggled('pd-q1', 'c-q1');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Checksum and make System Security 1 visible
+        assertHiddenAndVisibleToggled('c-q1', 'ss-q1');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 1 and make System Security 2 visible
+        assertHiddenAndVisibleToggled('ss-q1', 'ss-q2');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 2 and make System Security 3 visible
+        assertHiddenAndVisibleToggled('ss-q2', 'ss-q3');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 3 and make System Security 4 visible
+        assertHiddenAndVisibleToggled('ss-q3', 'ss-q4');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide System Security 4 and make Information Management 1 visible
+        assertHiddenAndVisibleToggled('ss-q4', 'im-q1');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Information Management 1 and make Information Management 2 visible
+        assertHiddenAndVisibleToggled('im-q1', 'im-q2');
+        cy.get('.box-body #questions-section #questions-buttons .button-row .btn-next').click({force: true})
+
+        // Hide Information Management 2 and make Information Management 3 visible
+        cy.get(`.box-body #questions-section #questions .question .question-content .question-text #im-q2`)
```

Comment:
You're using backticks here and for a few more lines below; I know it doesn't really have an effect but should we use quotes for consistency?

Reply:
that's probably just what intellij shown in autocomplete